mlrun/projects/project.py

```diff
@@ -210,15 +210,18 @@ def load_project(
 
     example::
 
-        # load the project and run the 'main' workflow
-        project = load_project("./", "git://github.com/mlrun/project-demo.git")
+        # load the project and run the 'main' workflow. The context directory must be an empty or 
+        # non-existent folder as the git repo will be cloned there
```

Comment:
I think this should be added under the url param
```suggestion
        # When using git as the url source the context directory must be an empty or non-existent folder as the git repo will be cloned there
```